tsdb/docs/format/head_chunks.md

```diff
@@ -0,0 +1,41 @@
+# Head Chunks on Disk Format
+
+The following describes the format of a chunks file,
+which is created in the `wal/chunks/` inside the data directory.
+
+Chunks in the files are referenced from the index by uint64 composed of
+in-file offset (lower 4 bytes) and segment sequence number (upper 4 bytes).
+
+The mint and maxt in the file header is the maximum and minimum times
+among all the chunks that it holds.
+
+```
+┌──────────────────────────────┐
+│  magic(0x85BD40DD) <4 byte>  │
```

Comment:
Just double-checking, I hope it is a different magic than our other files, right?

Reply:
I am re-using the same code to writing the initial headers of the segment files. So it will be the same.

Reply:
I think this is wrong. Magic code is the ID of the format. Since it's different than chunk segments, it has to have unique magic code IMO (:

Reply:
There is a version too for the file, so doesn't the version decide the format?

Reply:
No. Think of magic code as "Type" of format."Version" as instance of it.

https://en.wikipedia.org/wiki/Magic_number_(programming)

> A constant numerical or text value used to identify a file format or protocol; for files, see List of file signatures

Reply:
Okay. I will add a different magic number then. (This will be done in the new PR opened)